Makefile

```diff
@@ -95,6 +96,9 @@ move-other:
 lint:
 	@$(GOLANGCI_LINT) run -v --no-config --fast=false --fix --disable-all --enable goimports $(PACKAGE_PREFIX)
 
+fix-imports:
+	go run golang.org/x/tools/cmd/goimports@latest -l -w $(PACKAGE_PREFIX)
```

Comment:
I think we can get this for free with the `lint` task which already calls on `goimports` (and seemingly only `goimports`).

https://golangci-lint.run/usage/linters/#goimports 

We'd need a [.golangci.yaml](https://golangci-lint.run/usage/configuration/) with:
```yaml
linters-settings:
  goimports:
    local-prefixes: github.com/equinix-labs/metal-go
```

Reply:
It looks like we don't call `lint` in the current `all` chain (and it isn't marked as `PHONY`)

I checked if `gofumpt` (`fmt`) includes `goimports` behavior, but it appears not:
https://github.com/mvdan/gofumpt#frequently-asked-questions

> you should be able to call both tools; for example, goimports file.go && gofumpt file.go.

Reply:
I gave `golanci-lint` a whirl and it fails with this output:

Reply:
It looks like maybe I can't work around this? https://golangci-lint.run/usage/faq/#why-do-you-have-typecheck-errors

One of the things to check on that page if you get `typecheck` errors is:
> Ensure building works with go run ./.../go build ./... - whole package.

But in this case we're explicitly trying to make the code runnable/buildable by fixing missing imports.

Reply:
This is a moot point now, because it turns out that by upgrading `openapi-generator` to `v7.0.0` we got access to a generator option that will break inline enums out to separate models for us, so we don't need a custom template for this.  Latest commit on the PR converts from the custom template to the generator option.